CHANGELOG.md

```diff
@@ -4,8 +4,153 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [1.3.1](https://github.com/gradido/gradido/compare/1.2.1...1.3.1)
-
+#### [1.5.1](https://github.com/gradido/gradido/compare/1.5.0...1.5.1)
+
+> 15 October 2021
+
+- fix isExitInDb [`#994`](https://github.com/gradido/gradido/pull/994)
+- fix [`80228ef`](https://github.com/gradido/gradido/commit/80228ef842d4087ea4b80934b15b8112611e3e33)
+
+#### [1.5.0](https://github.com/gradido/gradido/compare/1.4.0...1.5.0)
+
+> 15 October 2021
+
+- v1.5.0 [`#986`](https://github.com/gradido/gradido/pull/986)
```

Comment:
It seems like that this is added after the creation of the changelog..
May be we could find something we have the release ticket in the changelog?

Reply:
The Changelog only displays pullrequests and no issues.
https://github.com/gradido/gradido/pull/986 is exactly the ticket where the version is set. We could link the release Ticket in there in the future.